VAGRANT.md

```diff
@@ -65,13 +61,31 @@ see Nominatim in action on [locahost:8089](http://localhost:8089/nominatim/).
 You edit code on your host machine in any editor you like. There is no need to
 restart any software: just refresh your browser window.
 
+Note that the webserver uses files from the /build directory. If you change
+files in Nominatim/lib for example you first need to copy them into the
+/build directory.
```

Comment:
Nominatim/lib is fine. /website and /utils are copied and they should be refreshed in the build directory by rerunning cmake.